docs/src/site/_layouts/default.html

```diff
@@ -24,9 +24,9 @@
         <div id="commute"><img src="{{ site.baseurl }}/img/circe.svg" width="100" height="100"></div>
         <div id="toolbox">
           <table id="tooltable">
-            <tr><td class="label">Source:</td><td class="link"><a href="https://github.com/travisbrown/circe">Github</a></td></tr>
+            <tr><td class="label">Source:</td><td class="link"><a href="https://github.com/travisbrown/circe">GitHub</a></td></tr>
             <tr><td class="label">Chat:</td><td class="link"><a href="https://gitter.im/travisbrown/circe">Gitter</a></td></tr>
-            <tr><td class="label">Build:</td><td class="link"><a href="https://travis-ci.org/travisbrown/circe">Travis</a></td></tr>
+            <tr><td class="label">Build:</td><td class="link"><a href="https://travis-ci.org/travisbrown/circe">Travis CI</a></td></tr>
```

Comment:
It must be quite inconvenient to share your name with a popular continuous integration service 😄